test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.Logging.DotNet6_0.verified.txt

```diff
@@ -78,6 +64,19 @@ WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();,
       },
       User: {},
       Spans: [
+        {
+          IsFinished: true,
+          Operation: db.connection,
+          Status: Ok,
+          IsSampled: true,
+          Extra: {
+            bytes_received: 677,
+            bytes_sent : 510,
+            db.connection_id: Guid_1,
+            db.operation_id: Guid_2,
+            rows_sent: 0
+          }
+        },
```

Comment:
Here you can see the connection (along with its statistics) where we weren't previously capturing a span for it.